Use a fully qualified hypertable name in an example (#732)
timescale/docs: timescaledb/how-to-guides/continuous-aggregates/materialized-hypertables.md

this materialized hypertable in the same way as any other hypertable.

## Discover the name of a materialized hypertable

> To change a materialized hypertable, you need to discover the fully qualified
> name of it. To do this, use the

Reviewer: ```suggestion
To change a materialized hypertable, you need to use its fully qualified
name. To find the correct name, use the
```
